hosting_docs/source/installation/migration/index.rst

```diff
@@ -0,0 +1,7 @@
+Migrating CommCareHQ
+====================
+
+This section describes how to migrate CommCareHQ data between different instances.
+
+* :doc:`/installation/migration/1-migrating-project` describes how to migrate a single project from one CommCareHQ instance to a new CommCareHQ instance.
```

Comment:
I didn't know about `:doc:`. I've been using `:ref:`. This is useful!

Reply:
Actually, I started using the style from https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html which is more powerful.